ProjectShop/UserApp/models.py

```diff
@@ -48,6 +51,8 @@ class User(AbstractBaseUser, PermissionsMixin):
     last_name = models.CharField(verbose_name=_('last name'), blank=False, null=False, max_length=40)
     middle_name = models.CharField(verbose_name=_('middle name'), blank=True, null=False, max_length=40)
 
+    profile_pic = models.ImageField(verbose_name=_('profile picture'), upload_to='profile_pictures/',
+                                    default='default_profile_pictures/default_pic.svg')
     birth_date = models.DateField(verbose_name=_('date of birth'), blank=True, null=True, auto_now=False,
                                   auto_now_add=False)
     register_date = models.DateField(verbose_name=_('date of registration'), blank=False, null=False,
```

Comment:
There is no need to store high-resolution profile pictures so that it might be useful to add the following:
def save(self, *args, **kwargs):
    """Resize a profile picture to a smaller size."""
    super().save(*args, **kwargs)
   
    profile_pic = profile_pic.open(self.profile_pic.path)
   
    if profile_pic.height > 300 or profile_pic.width > 300:
        output_size = (300, 300)
        img.thumbnail(output_size)
        img.save(self.profile_pic.path)